src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+}
+
+div .stars__star:not(:first-child) {
```

Comment:
Get used to style all elements using classes. And don't increase selectors specificity unless completely necessary